tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java

```diff
@@ -192,19 +194,30 @@ private String getElementOrAttributeName(QName qName) {
      * the intermediate object stack
      *
      * @param attributes List of attributes associate with a mule element
-     * @param mClass     Intermediate class that is mapped to the mule element
+     * @param mClassName Intermediate class that is mapped to the mule element
      */
-    private void populateIntermediateObject(Iterator<Attribute> attributes, Class<?> mClass) {
+    private void populateIntermediateObject(Iterator<Attribute> attributes, String mClassName, String elementName) {
         try {
-            java.lang.Object object = mClass.newInstance();
+            java.lang.Object object = null;
```

Comment:
Any reason for using fully qualified classes here ?

Reply:
Replaced with import. https://github.com/wso2/product-ei/pull/1216/commits/5bc332461335b908d9a95df22ffdb43bd8ebe7c0